community/cypher/cypher-compiler-3.0/src/main/scala/org/neo4j/cypher/internal/compiler/v3_0/executionplan/CompiledExecutionResult.scala

```diff
@@ -271,5 +55,9 @@ class CompiledExecutionResult(taskCloser: TaskCloser,
 
     compiledCode.executionPlanDescription()
   }
-}
 
+  override def queryStatistics() = InternalQueryStatistics()
+
+  //TODO delegate to compiled code once writes are being implemented
+  override def executionType = READ_ONLY
```

Comment:
There will soon be coming actual procedures that also does writes, the `executionType` will not be a problem, not exactly sure about `queryStatistics` though bet let's worry about that when we get to updating procedures. So far all procedures are read-only.